donor-portal/package.json

```
@@ -25,7 +25,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "ESLINT_NO_DEV_ERRORS=true react-scripts start",
+    "start": "react-scripts start",
```

Can you keep this the way it was? @Sanjana0319 
```suggestion
    "start": "ESLINT_NO_DEV_ERRORS=true react-scripts start",
```